### PR TITLE
Increase gap between bars

### PR DESCRIPTION
**Pull request by okal**

### CODE REVIEW

File: src/vega_viewer/vega/specs.cljs [hunk 1]
```diff
@@ -10,23 +10,27 @@
    :scales [{:name "category"
              :type "ordinal"
              :domain {:data "entries" :field "category"}
-             :bandWidth bar-height
+             :bandWidth (+ 3 bar-height)
```
Comment (pld) [inline src/vega_viewer/vega/specs.cljs]: `3` move to a constant?

File: src/vega_viewer/vega/specs.cljs [hunk 2]
```diff
@@ -10,23 +10,27 @@
...
+           :properties {:labels {:text {:template "{{datum.data}}"}}}}
           {:scale "category" :type "y"}]
    :marks [{:from {:data "entries"}
             :type "rect"
-            :properties {:enter {:y {:scale "category" :field "category"}
-                                 :height {:value 30
-                                          :offset -1}
+            :properties {:enter {:y {:scale "category"
+                                     :field "category"
+                                     :offset 3}
```
Comment (pld) [inline src/vega_viewer/vega/specs.cljs]: is the `3` up there the same as this `3`? or a coincidence?

File: src/vega_viewer/vega/specs.cljs [hunk 3]
```diff
@@ -10,23 +10,27 @@
...
+                                 :height {:value bar-height
+                                          :offset -1.5}
```
Comment (pld) [inline src/vega_viewer/vega/specs.cljs]: also to a constant

Comment (pld) [inline src/vega_viewer/vega/specs.cljs]: or maybe not, there's a lot of constanty things, you can decide what makes the cut, anything duplicated, reused, likely to be tweaked, or easier to explain with a descriptive name should go a constant

Comment (okal) [inline src/vega_viewer/vega/specs.cljs]: Yeah, this should be a constant. Was just messing with values. This should have been a DNM. I'll fix it.

**Pull request merged**